src/vendor/webauthn.js

```diff
@@ -0,0 +1,185 @@
+"use strict";
```

Comment:
The file is coming from the [Webauth polyfill](https://github.com/adrianba/fido-snippets/blob/master/polyfill/webauthn.js) taken from the [dev microsoft page](https://developer.microsoft.com/en-us/microsoft-edge/platform/documentation/dev-guide/device/web-authentication/).
It doesn't look like a good solution to get it from npm.

The file was moved to `utils/webauth.js` and refactored.